### PR TITLE
Auto update mimir dashboards

### PR DESCRIPTION
**Pull request by QuentinBisson**
This PR

- adds a script to update the mimir dashboards https://github.com/giantswarm/giantswarm/issues/30310

### Checklist

- [x] Update changelog in CHANGELOG.md in an end-user friendly language.

### CODE REVIEW

**Comment by QuentinBisson** [comment]
We do have an issue which I added in the top description. I'm aware of the changes and I will check what we can to do add them here, either replacing them using jq, fix them properly upstream or in our setup :)

**Comment by QuentinBisson** [comment]
I removed the loki changes, those were because my main branch was outdated

**Review by QuantumEnigmaa — APPROVED** [review]
LGTM

**Comment by QuentinBisson** [comment]
Current status is described here https://github.com/giantswarm/giantswarm/issues/30310#issuecomment-2062481773

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,6 +23,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Move `node-problem-detector` to be aws only.
 
+
```
Comment (QuentinBisson) [inline CHANGELOG.md:26]:
```suggestion
```

**Comment by QuentinBisson** [comment]
All dashboards are working as expected, merging 🎉

**Pull request merged**